app/Http/Controllers/ViewController.php

```diff
@@ -1,5 +1,7 @@
 <?php
 
+/** @noinspection PhpUndefinedClassInspection */
```

Comment:
I really don't like those lines. The suppress which potential errors which are otherwise reported by a proper IDE. Please import the `Lang` face via a proper `use`-statement.